lib/scheduleV2/ScheduleV2.js

```diff
@@ -12,74 +12,84 @@ function scheduleV2()
+      create: function (data, config, cb) {
+        api.post(baseURL, data, config, cb);
+      },
 
-        /**
-         * https://docs.opsgenie.com/docs/schedule-api#section-create-schedule
-         * @param data
-         * @param config
-         * @param cb
-         */
-        create: function (data, config, cb) {
-            api.post(baseURL, data, config, cb)
-        },
+      /**
+       * https://docs.opsgenie.com/docs/schedule-api#section-delete-schedule
+       * @param identifier
+       * @param config
+       * @param cb
+       */
+      delete: function (identifier, config, cb) {
+        let path = api.getPath(baseURL, identifier, null);
+        api.delete(path, config, cb);
+      },
 
-        /**
-         * https://docs.opsgenie.com/docs/schedule-api#section-delete-schedule
-         * @param identifier
-         * @param config
-         * @param cb
-         */
-        delete: function (identifier, config, cb) {
-            let path = api.getPath(baseURL, identifier, null);
-            api.delete(path, config, cb)
-        },
+      /**
+       * https://docs.opsgenie.com/docs/schedule-api#section-update-schedule-partial
+       * @param identifier
+       * @param data
+       * @param config
+       * @param cb
+       */
+      updateSchedule: function (identifier, data, config, cb) {
+        let path = api.getPath(baseURL, identifier, null);
+        api.patch(path, data, config, cb);
+      },
 
-        /**
-         * https://docs.opsgenie.com/docs/schedule-api#section-update-schedule-partial
-         * @param identifier
-         * @param data
-         * @param config
-         * @param cb
-         */
-        updateSchedule: function (identifier, data, config, cb) {
-            let path = api.getPath(baseURL, identifier, null);
-            api.patch(path, data, config, cb)
-        },
+      /**
+       * https://docs.opsgenie.com/docs/schedule-api#section-get-schedule-timeline
+       * @param identifier
+       * @param config
+       * @param cb
+       */
+      getScheduleTimeline: function (identifier, config, cb) {
+        let path = api.getPath(baseURL, identifier, timeline);
+        let finalPath = api.getPathWithListParams(path, config);
+        api.get(finalPath, config, cb);
+      },
 
-        /**
-         * https://docs.opsgenie.com/docs/schedule-api#section-get-schedule-timeline
-         * @param identifier
-         * @param config
-         * @param cb
-         */
-        getScheduleTimeline: function (identifier, config, cb) {
-            let path = api.getPath(baseURL, identifier, timeline);
-            let finalPath = api.getPathWithListParams(path, config);
-            api.get(finalPath, config, cb)
-        },
-    }
+      /**
+       * https://docs.opsgenie.com/docs/who-is-on-call-api
+       * @param identifier
+       * @param cb
+       */
+      getOnCallForSchedule: function (identifier, cb) {
+        let path = api.getPath(baseURL, identifier, onCalls);
```

Comment:
I've made a small change here as the action path is `on-calls` https://docs.opsgenie.com/docs/who-is-on-call-api#get-on-calls